LongWarOfTheChosen/Config/XComLW_FactionBalance.ini

```diff
@@ -86,11 +89,19 @@ OVERRIDE_REDUCTION=10
 
 [LW_FactionBalance.X2LWModTemplate_ReaperAbilities]
 SHADOW_DURATION=1
-SHADOW_DETECTION_RANGE_REDUCTION=0.7f
+SHADOW_DETECTION_RANGE_REDUCTION=0.9f
 
 REMOTE_START_CHARGES=1
 REMOTE_START_DEMOLITIONIST_CHARGES=1
 
+BLOOD_TRAIL_ANTIDODGE_BONUS=20

```

Comment:
Since the dodge malus here can't go below 0 to boost crit, I wonder if it'd be better to make the malus bigger, after all it does not affect graze band and mathematicaly 20 dodge is 10% dmg reduction. I'd try 30 or even 40 to make it feel more impactful even if most targets don't even have that much dodge.

Reply:
Sure. Let's go ham and change it to 40.